cmd/network-operator-init-container/app/app.go

```diff
@@ -128,6 +116,30 @@ func RunNetworkOperatorInitContainer(ctx context.Context, config *rest.Config, o
 		return err
 	}
 
+	confConfigMap := &corev1.ConfigMap{}
+
+	err = k8sClient.Get(ctx, client.ObjectKey{
+		Name:      opts.ConfigMapName,
+		Namespace: opts.ConfigMapNamespace,
+	}, confConfigMap)
+
+	if err != nil {
+		logger.Error(err, "failed to read config map with configuration")
+		return err
+	}
+
+	initContCfg, err := configPgk.Load(confConfigMap.Data[opts.ConfigMapKey])
+	if err != nil {
+		logger.Error(err, "failed to read configuration")
+		return err
+	}
+	logger.Info("network-operator-init-container configuration", "config", initContCfg.String())
+
+	if !initContCfg.SafeDriverLoad.Enable {
```

Comment:
any use-case where we disable safe driver load in runtime ?

Reply:
Do you mean we should watch for changes in ConfigMap and cancel waiting if safeLoad is disabled? I think we don't need this. Such logic will complicate things and may impact new features we may want to add to the init container. Not all features in the future may play nicely with cancellation logic. 

 If the user will need to cancel waiting, then we can propose to remove safe driver loading label manually from the node object - this will unblock loading instantly.

Reply:
ack